File: 백준/Silver/1427. 소트인사이드/소트인사이드.java
import java.util.Scanner;

public class Main {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        String s = sc.next();
        int n = s.length();
        int[] nums = new int[n];
        for (int i = 0; i < n; i++) {
            nums[i] = s.charAt(i) - '0';
        }
        
        /* Selection Sort */
        for (int i = 0; i < n; i++) {
            int max = i;
            for (int j = i + 1; j < n; j++) {
                if (nums[j] > nums[max]) max = j;
            }
            
            if (nums[i] < nums[max]) {
                int tmp = nums[i];
                nums[i] = nums[max];
                nums[max] = tmp;
            }
        }
        
        for (int i = 0; i < n; i++) {
            System.out.print(nums[i]);
        }
    }
}

File: 백준/Silver/1427. 소트인사이드/소트인사이드.kt
import java.util.Scanner

fun main() {
    val sc = Scanner(System.`in`)
    val s = sc.next()
    val n = s.length
    val nums = IntArray(n)
    
    for (i in 0 until n) {
        nums[i] = s[i] - '0'
    }
    
    /* Selection Sort */
    for (i in 0 until n) {
        var max = i
        for (j in i + 1 until n) {
            if (nums[j] > nums[max]) max = j
        }
        
        if (nums[i] < nums[max]) {
            nums[i] = nums[max].also { nums[max] = nums[i] }
        }
    }
    
    for (i in 0 until n) {
        print(nums[i])
    }
}